src/_P036_FrameOLED.ino

```diff
@@ -368,7 +422,8 @@ boolean Plugin_036(byte function, struct EventStruct *event, String& string)
       {
         lastWiFiState = P36_WIFI_STATE_UNSET;
         // Load the custom settings from flash
-        Plugin_036_loadDisplayLines(event->TaskIndex);
+        byte version = (Settings.TaskDevicePluginConfigLong[event->TaskIndex][0] & 0xF00000) >> 20;    // Bit23-20
```

Comment:
Please make a function to decode values like these.
A cut/paste error is easy to make here.

Reply:
Done!